lib/miq_automation_engine/engine/miq_ae_engine/drb_remote_invoker.rb

```diff
@@ -45,20 +45,20 @@ def teardown
       global_id_conv = drb_server.config[:idconv]
       drb_server.stop_service
       self.drb_server = nil
+
       # This hack was done to prevent ruby from leaking the
       # TimerIdConv thread.
       # https://bugs.ruby-lang.org/issues/12342 (has been fixed in ruby 2.4.0 preview 1)
       # also fixed in ruby_2_3 branch for the 2.3.2 release: https://github.com/ruby/ruby/commit/c20b07d5357d7cb73226b149431a658cde54a697
-      thread = global_id_conv
-               .try(:instance_variable_get, '@holder')
-               .try(:instance_variable_get, '@keeper')
-      if RUBY_VERSION > "2.3.1"
-        warn "Remove me: #{__FILE__}:#{__LINE__} and my test. Ruby 2.3.2+ should not be creating timer threads."
```

Comment:
In an ideal world, we would open a PR now to remove it, which would fail tests now but start passing as soon as .travis.yml tests only >2.3.1.
But something would have to periodically re-run the tests and notify someone once they start passing.  Or directly "merge when green", measured in years :trollface:
> "An ideal world is left as an exercise to the reader" — Paul Graham,  _On Lisp_

Silly question:
We're bumping to requiring ≥2.3 now that we'll support 2.4, no?
Can't we simply proclaim we support ≥2.3.2 and drop the hack?